Sources/IndicatorInfo.swift

```diff
@@ -26,21 +26,13 @@ import Foundation
 
 public struct IndicatorInfo {
     
-    public var title: String
+    public var title: String?
     public var image: UIImage?
     public var highlightedImage: UIImage?
     
-    public init(title: String) {
+    public init(title: String? = nil, image: UIImage? = nil, highlightedImage: UIImage? = nil) {
```

Comment:
@fedeojeda95  I agree with everything you said & we should add this pr to our backlog.
Nevertheless, i think we should merge this PR as is in order to fix the missing outlet issues.